src/screens/ConfirmDepositPix.tsx

```diff
@@ -42,70 +44,85 @@ export default function ConfirmDepositPix({
+          <WarningCard text="Deposite de uma conta com o mesmo CPF ou CNPJ da sua conta. Não são aceitos depósitos de terceiros." />
 
-        <View style={styles.item}>
-          <Label size="sm-regular" style={styles.labelSmall}>
-            Enviado
-          </Label>
+          <View style={styles.item}>
+            <Label size="sm-regular" style={styles.labelSmall}>
+              Enviado
+            </Label>
 
-          <Heading>
-            {amount.toLocaleString("pt-BR", CURRENCYFORMAT)} BRL
-          </Heading>
-        </View>
+            <Heading>
+              {amount.toLocaleString("pt-BR", CURRENCYFORMAT)} BRL
+            </Heading>
+          </View>
 
-        <View style={styles.item}>
-          <Label size="sm-regular" style={styles.labelSmall}>
-            Taxas
-          </Label>
+          <View style={styles.item}>
+            <Label size="sm-regular" style={styles.labelSmall}>
+              Taxas
+            </Label>
 
-          <Label>0,5% Transfero</Label>
-        </View>
+            <Label>0,5% Transfero</Label>
+          </View>
 
-        <View style={styles.item}>
-          <Label size="sm-regular" style={styles.labelSmall}>
-            Total a ser recebido
-          </Label>
+          <View style={styles.item}>
+            <Label size="sm-regular" style={styles.labelSmall}>
+              Total a ser recebido
+            </Label>
 
-          <View style={styles.symbolContainer}>
-            <View style={styles.imageWrapper}>
-              <Image style={styles.image} source={image} contentFit="cover" />
+            <View style={styles.symbolContainer}>
+              <View style={styles.imageWrapper}>
+                <Image style={styles.image} source={image} contentFit="cover" />
+              </View>
+
+              <Heading size="heading2">
+                {price.toLocaleString("pt-BR", SIMPLECURRENCYFORMAT)} {currency}
+              </Heading>
             </View>
 
-            <Heading size="heading2">
-              {price.toLocaleString("pt-BR", SIMPLECURRENCYFORMAT)} {currency}
-            </Heading>
+            <Paragraph size="xs" style={styles.labelSmall}>
+              Esse valor é válido até{" "}
+              {date.toLocaleTimeString("pt-BR", {
+                hour: "numeric",
+                minute: "numeric"
+              })}
+            </Paragraph>
           </View>
 
-          <Paragraph size="xs" style={styles.labelSmall}>
-            Esse valor é válido até{" "}
-            {date.toLocaleTimeString("pt-BR", {
-              hour: "numeric",
-              minute: "numeric"
-            })}
-          </Paragraph>
-        </View>
-
-        <View style={styles.item}>
-          <Label size="sm-regular" style={styles.labelSmall}>
-            Chave pix Transfero
-          </Label>
+          <View style={styles.item}>
+            <Label size="sm-regular" style={styles.labelSmall}>
+              Chave pix Transfero
```

Comment:
Shouldn't "pix" be capitalized as "PIX"?

```suggestion
              Chave PIX Transfero
```